src/persist-client/src/cli/admin.rs

```diff
@@ -671,7 +671,7 @@ async fn force_gc(
 /// Exposed for `mz-catalog`.
 pub const CATALOG_FORCE_COMPACTION_FUEL: Config<usize> = Config::new(
     "persist_catalog_force_compaction_fuel",
-    0,
+    1024,
```

Comment:
did you mean to include this change in the PR?

Reply:
Yeah, I'll update the commit description. It's in the linked issue.